src/main/java/com/crud/library/controller/RentalController.java

```diff
@@ -0,0 +1,37 @@
+package com.crud.library.controller;
+
+import com.crud.library.dbService.RentalService;
+import com.crud.library.exception.ValueAlreadyExistsException;
+import com.crud.library.exception.ValueNotFoundException;
+import com.crud.library.domain.Rental;
+import com.crud.library.domain.RentalDto;
+import com.crud.library.mapper.RentalMapper;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/library/rental")
+@CrossOrigin(origins = "*")
+@RequiredArgsConstructor
+public class RentalController {
+    private final RentalService rentalService;
+    private final RentalMapper rentalMapper;
+
+    @PostMapping(value = "createRental")
```

Comment:
nazwy mappingów powinny być krótkie, masz w linii 15 powiedziane ze to jest rental, a wiec wystarczy tu create, nizej get